File: 0601_0700/0643/0643.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var n int
	fmt.Fscan(f, &n)
	for i := n; i != 0; i &= i - 1 {
		n++
	}
	fmt.Fprint(q, n)
}

File: 0601_0700/0643/0643.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n;

	fscanf(f, "%d", &n);
	for (int i = n; i; i &= i - 1)
		++n;
	fprintf(q, "%d", n);
	return 0;
}
